src/annotator/anchoring/pdf.ts

```diff
@@ -626,13 +626,18 @@ export async function anchor(
   }
 }
 
+function clamp(x: number, min: number, max: number): number {
+  return Math.max(Math.min(x, max), min);
+}
```

Comment:
Perhaps `value` instead of `x`?

```suggestion
function clamp(value: number, min: number, max: number): number {
  return Math.max(Math.min(value, max), min);
}
```

Reply:
I don't have a strong preference here, but `x` is the convention used in eg. the [`Math` function docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs).

Reply:
Oh! Then `x` makes sense. I though it might have been a side effect of copy-pasting an older function where `x` and/or `y` was used.